src/shared/constants/dimensions.js

```diff
@@ -7,27 +7,16 @@ export const HEIGHT_BREAKPOINT_2 = 230
 export const HEIGHT_BREAKPOINT_3 = 285
 export const AXIS_TICK_WIDTH = 8
 export const AXIS_TICK_PADDING = 8
-/**
- * BOTTOM_LEGEND_ADJUSTMENT - was defined to adjust the x-axis legend distance to the x-axis
- * tick labels (which should be 8px), so the legend sits exactly the same distance that the
- * y-axis legend is positioned relative to the y-axis tick labels. This adjustment is needed to
- * solve some of the limitations found when implementing Nivo charts to fit design specifications.
- */
-export const BOTTOM_LEGEND_ADJUSTMENT = 8
-export const LEGEND_HEIGHT = 17
-export const TEXT_HEIGHT = 17
 export const BUFFER = 8
 export const SYMBOL_SIZE = 8
 export const SYMBOL_SPACING = 6
-export const FONT_SIZE = 12
 // minimum width in pixels of the legend keys when we start trimming if necessary
 export const TRIMMED_LEGEND_WIDTH = 42
 export const LEGEND_TRANSLATE_X = 14
-export const LEGEND_TRANSLATE_Y = LEGEND_HEIGHT + 2 * BUFFER
+export const LEGEND_BUFFER = 8
```

Comment:
_pref:_ just use BUFFER for consistency

Reply:
Sure, also BUFFER and LEGEND_BUFFER ha ve no relationship to each other. It just happens they have the same value, LEGEND_BUFFER is an empirical value, based on my tests.